artifactory_test.go

```diff
@@ -3142,12 +3145,36 @@ func TestArtifactorySearchIncludeDir(t *testing.T) {
 	// Search without IncludeDirs
 	searchCmd.SetSpec(searchSpecBuilder.IncludeDirs(false).BuildSpec())
 	assert.NoError(t, searchCmd.Search())
-	assert.ElementsMatch(t, searchCmd.SearchResult(), tests.GetSearchNotIncludeDirsFiles())
+	assert.ElementsMatch(t, searchCmd.SearchResultNoDate(), tests.GetSearchNotIncludeDirsFiles())
 
 	// Search with IncludeDirs
 	searchCmd.SetSpec(searchSpecBuilder.IncludeDirs(true).BuildSpec())
 	assert.NoError(t, searchCmd.Search())
-	assert.ElementsMatch(t, searchCmd.SearchResult(), tests.GetSearchIncludeDirsFiles())
+	assert.ElementsMatch(t, searchCmd.SearchResultNoDate(), tests.GetSearchIncludeDirsFiles())
+
+	// Cleanup
+	cleanArtifactoryTest()
+}
+
+func TestArtifactorySearchDisplayDate(t *testing.T) {
```

Comment:
To save execution time, I suggest removing this test, and instead, tale the following two lines from this test:
```
	assert.NotEmpty(t, result[0].Created)
	assert.NotEmpty(t, result[0].Modified)
```
and add them to all the existing search tests functions.

Reply:
Done.